docs/source/reference/api-server/examples/api-server-metrics-setup.rst

```diff
@@ -35,6 +35,21 @@ chart deploy everything for you with a single command:
       --set prometheus.enabled=true \
       --set grafana.enabled=true
 
+.. note::
+
+   The above command also configures Prometheus to scrape the SkyPilot API server's ``/gpu-metrics`` endpoint. To disable scraping of ``/gpu-metrics``, append ``--set prometheus.extraScrapeConfigs=""`` to the Helm command:
+
+   .. code-block:: bash
+
+       helm upgrade --install skypilot skypilot/skypilot-nightly --devel \
+         --namespace skypilot \
+         --create-namespace \
+         --reuse-values \
+         --set apiService.metrics.enabled=true \
+         --set prometheus.enabled=true \
+         --set prometheus.extraScrapeConfigs="" \
+         --set grafana.enabled=true
+
```

Comment:
Make this a dropdown instead? Most of the time, people should not need to turn it off.

Reply:
Updated